terraform/modules/core/output.tf

```diff
@@ -17,3 +17,11 @@ output "default_security_group_id" {
 output "ecs_autoscaling_group_name" {
   value = "${aws_autoscaling_group.ecs-autoscaling-group.name}"
 }
+
+output "private_subnet_ids" {
+  value = ["${aws_subnet.private_subnet.*.id}"]
+}
+
+output "public_subnet_ids" {
```

Comment:
We add this as an output but I don't see that we're using it so looks like it can safely be trimmed before it goes in.

Reply:
From what I've seen with other 3rd party modules, outputs should be added when modules are created, as we may need to use them in future.

Reply:
The principle of YAGNI (you ain't going to need it) is a useful one to bear in mind in these types of situation as it helps keep the code base simpler and easier to comprehend. Ignored enough times things can quickly get out of control and fill up with code we never even used.

However as this is an external module that we've copied across for connivence (which I think was a good balance to strike) we can leave it on the basis that we don't 'own it' even though it's in our codebase, someone who misses this fact in future might change things anyway thinking they're helping and tidying things up! 

Worth always thinking about.